webplayer-example-next/README.md

```diff
@@ -81,7 +81,7 @@ const response = await fetch(LOGIN_URL, {
   body: JSON.stringify({
     instanceId,
     projectId,
-    expiresIn: 18000,
+    expiresIn: 18000, // value is in seconds, 18000 = 5 hours
```

Comment:
It can be easier to understand if you write these values as 60 * 60 * 5 instead of the final number